src/System.Security.Cryptography.Encoding/tests/Base64TransformsTests.cs

```diff
@@ -43,6 +43,13 @@ public static IEnumerable<object[]> TestData_Ascii_Whitespace()
             yield return new object[] { "foo", " Z m 9 v" };
         }
 
+        public static IEnumerable<object[]> TestData_Oversize()
+        {
+            yield return new object[] { "Zm9v////", 0, 4, "foo" };
+            yield return new object[] { "////Zm9v", 4, 4, "foo" };
+            yield return new object[] { "////Zm9v////", 4, 4, "foo" };
```

Comment:
Why so many slashes?  If they're needed that suggests that something is looking at the array length when it should be looking at inputCount.

Reply:
I tried with only one slash but since I didn't call `TransformFinalBuffer` and the extra bytes on the end of the buffer didn't add up to at least one block, the extra slash would just get copied into the buffer by `TransformBuffer` so the test would still pass even without the fix. Maybe I should have only used `TransformFinalBuffer`.

Reply:
If it's reading past `inputCount` that's a bug that should be fixed.  (The only case I knew if where it was doing that should have been fixed with this change)

Reply:
And actually I think even if I were calling TransformFinalBlock I would still need at least four additional bytes due to the way incomplete blocks are handled.